### ISSUE TITLE
How to install Sulley on Linux?

### ISSUE THREAD

**Issue by hsw247119194** [OpenRCE/sulley#108]
Hello,
I have finished installed Sulley on windows10 or windows xp , but how can I install it on Linux?

Regards.

**Comment by q2dg** [comment]
It seems this project is pretty abandoned

**Comment by jtpereyda** [comment]
I support a fork https://github.com/jtpereyda/boofuzz

**Comment by Fitblip** [comment]
Yeah at this point Sulley is in nothing more than maintenance mode since I simply don't have time to dedicate to it. Use @jtpereyda's boofuzz (though I personally use AFL for my fuzzing needs now a days).
